src/ProductConstructionService/ProductConstructionService.Api/Queue/JobProcessorScopeManager.cs

```diff
@@ -10,15 +10,17 @@ public enum JobsProcessorState
     // The JobsProcessor isn't doing anything
     Stopped,
     // The JobsProcessor will finish its current job and stop
-    Stopping
+    Stopping,
+    // The JobsProcessor is waiting for VMR cloning to finish
+    WaitingForVmrClone
```

Comment:
I'd go more generic and call it `Starting` so that we can hide behind it whatever we will need to happen.

Reply:
I would not couple it too much with the VMR. It should be coupled with the fact we have some unspecified start up steps we need before we're ready. Applies to `VmrCloneDone` method too.

Reply:
since we're making this initialization generic, I'll go ahead and rename the background service too, it will be `InitializationBackgroundService` and we can add whatever we want there